docker-compose.yml

```
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  db:
+    image: postgres:13.1
```

```suggestion
    image: scalingo/postgresql:12.5.0-1
```

la version utilisée par scalingo de PG, j'ai activé une sandbox PG sur le staging
 

- https://db-osc-fr1.scalingo.com/dashboard/60254fbb987e0b73291c4fa7/overview
- https://hub.docker.com/layers/scalingo/postgresql/12.5.0-1/images/sha256-3e38d2873b5c8d78741b428982d20cb129c534fb93b4a7edc28d529e4726cce0?context=explore